### ISSUE TITLE
Video\Clip Download TypeError: 'NoneType' object is not subscriptable 

### ISSUE THREAD

**Issue by brunommpreto** [ihabunek/twitch-dl#87]
## Python Host
```

NAME="openSUSE Tumbleweed"
# VERSION="20220128"
ID="opensuse-tumbleweed"
ID_LIKE="opensuse suse"
VERSION_ID="20220128"
PRETTY_NAME="openSUSE Tumbleweed"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:tumbleweed:20220128"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
DOCUMENTATION_URL="https://en.opensuse.org/Portal:Tumbleweed"
LOGO="distributor-logo-Tumbleweed"

```

## Python Version

```

Python 3.9.7
```    

## Video Download Log
```
Looking up video...
Found: $title by $author
Traceback (most recent call last):
  File "/home/brunommpreto/.local/bin/twitch-dl", line 8, in <module>
    sys.exit(main())
  File "/home/brunommpreto/.local/pipx/venvs/twitch-dl/lib64/python3.8/site-packages/twitchdl/console.py", line 243, in main
    args.func(args)
  File "/home/brunommpreto/.local/pipx/venvs/twitch-dl/lib64/python3.8/site-packages/twitchdl/commands/download.py", line 156, in download
    return _download_video(video_id, args)
  File "/home/brunommpreto/.local/pipx/venvs/twitch-dl/lib64/python3.8/site-packages/twitchdl/commands/download.py", line 255, in _download_video
    target = _video_target_filename(video, args)
  File "/home/brunommpreto/.local/pipx/venvs/twitch-dl/lib64/python3.8/site-packages/twitchdl/commands/download.py", line 82, in _video_target_filename
    "game": video["game"]["name"],
TypeError: 'NoneType' object is not subscriptable

```

## Clip Download Log
```
Looking up clip...
Traceback (most recent call last):
  File "/home/brunommpreto/.local/bin/twitch-dl", line 8, in <module>
    sys.exit(main())
  File "/home/brunommpreto/.local/pipx/venvs/twitch-dl/lib64/python3.8/site-packages/twitchdl/console.py", line 243, in main
    args.func(args)
  File "/home/brunommpreto/.local/pipx/venvs/twitch-dl/lib64/python3.8/site-packages/twitchdl/commands/download.py", line 160, in download
    return _download_clip(clip_slug, args)
  File "/home/brunommpreto/.local/pipx/venvs/twitch-dl/lib64/python3.8/site-packages/twitchdl/commands/download.py", line 220, in _download_clip
    clip["game"]["name"],
TypeError: 'NoneType' object is not subscriptable

```

"TypeError: 'NoneType' object is not subscriptable"

**Comment by ihabunek** [comment]
What version of twitch-dl are you using? You probably need to upgrade.

**Comment by brunommpreto** [comment]
twitch-dl v1.18.0

**Comment by ihabunek** [comment]
Fixed in [1.18.1](https://github.com/ihabunek/twitch-dl/releases/tag/1.18.1).
